lib/symengine.rb

```diff
@@ -33,6 +33,29 @@ def Function(n)
     def evalf(operand, prec=53, real=false)
         return _evalf(operand, prec, real)
     end
+    def evalf_dirichlet_eta(exp)
+        SymEngine::evalf(SymEngine::dirichlet_eta(exp))
+    end
+    def evalf_zeta(exp)
+        SymEngine::evalf(SymEngine::zeta(exp))
+    end
+    def lambdify_code(exp)
+      sym_map = exp.free_symbols.map { |sym| sym.to_s}.sort.join(",")
```

Comment:
You sometimes need the order of the variable and this doesn't give you that.
For the 2nd part an eg:
```
ex = expand((x + y)**2 - 2*x*y-y**2)
l = lambdify(ex)
l.call(1, 2)
```
User expects in the last call that the expression is evaluated at x=1 and y=2, but this would give an error because there is only one free symbol in ex, which is x